notification/webhook/sender.go

```diff
@@ -0,0 +1,101 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/target/goalert/notification"
+)
+
+type Sender struct{}
+
+type WebhookAlert struct {
```

Comment:
Generally, it's best to avoid "stutters" in names, this will be used as `webhook.WebhookAlert` maybe `POSTData` or `NotificationData` -- something descriptive to its use. Also, add comments for exported names.